### DISCUSSION TITLE
GitHub organizations premium for a student group

### DISCUSSION THREAD

**Discussion by susanthenerd** [community/community#42277] [GitHub Education]
Hello,
I'm part of our [robotics team](https://github.com/kronbot) in our high school and we would love to have premium on our github organization. Is it possible? If yes what information's do we have to provide.

Thanks,
Cristian

**Comment by samus-aran** [comment]
Hi @cristiioan 👋 

That's a great question. Usually a faculty member has the ability to upgrade an organisation once they've been verified. It would be awesome to hear what you plan on using with the organisation within your robotics team!

**Reply by susanthenerd** [reply]
So it is not possible to get premium for our org if we are in high school?
We are using the github organization for easy colaboration on code for the First Tech Chalange.
